nodes/docker-images.md

```diff
@@ -76,8 +76,9 @@ Ubuntu. You can
 
    :::
 
-3. Set an RPC endpoint for either [Arabica](./arabica-devnet.md#rpc-endpoints) or
-   [Mocha](./mocha-testnet.md#rpc-endpoints)
+3. Set an RPC endpoint for either [Mainnet Beta](./mainnet.md#da-rpc-endpoints),
+   [Mocha](./mocha-testnet.md#rpc-endpoints), or
+   [Arabica](./arabica-devnet.md#rpc-endpoints)
    using the bare URL (without http or https):
 
    ```bash
```

Comment:
It feels like this next line would ideally provide a sample command with a real RPC_URL (or a few different ones) across the different networks as is done for the other commands (e.g. 4 from below let's you choose the network you are trying to run on and provides the sample command to use in that case).

Reply:
i will not provide a sample command with a sample URL as to allow the user to pick their own